datahost-ld-openapi/src/tpximpact/datahost/ldapi/routes/revision.clj

```diff
@@ -1,11 +1,13 @@
 (ns tpximpact.datahost.ldapi.routes.revision
   (:require
     [reitit.ring.malli]
+    [reitit.coercion.malli :as rcm]
     [tpximpact.datahost.ldapi.handlers :as handlers]
     [tpximpact.datahost.ldapi.routes.shared :as routes-shared]))
 
 (defn get-revision-route-config [db]
   {:summary "Retrieve metadata for an existing revision"
+   :coercion (rcm/create {:transformers {}, :validate false})
```

Comment:
This is the actual "fix" for now. The coercion middleware is looking for a registered transformer and there isn't one yet; I'm not sure we need one either.